src/test/scala-2.12/com/linagora/gatling/imap/protocol/ImapSessionsSpec.scala

```diff
@@ -16,25 +17,26 @@ class ImapSessionsSpec extends WordSpec with Matchers with ImapTestUtils with Be
   private val server: RunningServer = CyrusServer.start()
 
   override def beforeEach(): Unit = {
-    server.addUser("user1", "password")
+    server.addUser(bart)
   }
 
   override protected def afterEach(): Unit = {
     system.terminate()
     server.stop()
   }
 
-  implicit lazy val system = ActorSystem("LoginHandlerSpec")
+  implicit lazy val system: ActorSystem = ActorSystem("LoginHandlerSpec")
   "the imap sessions actor" should {
     "log a user in" in {
       val config = new Properties()
       val protocol = ImapProtocol("localhost", port = server.mappedImapPort(), config = config)
 
       val sessions = system.actorOf(ImapSessions.props(protocol))
       val probe = TestProbe()
-      probe.send(sessions, Command.Connect("1"))
+      val userId = "1"
+      probe.send(sessions, Command.Connect(userId))
       probe.expectMsg(10.second, Response.Connected(ImapResponses.empty))
-      probe.send(sessions, Command.Login("1", "user1", "password"))
+      probe.send(sessions, Command.Login(userId, bart))
```

Comment:
what's the purpose of this userId?

Reply:
if I remember correctly, it creates an actor with this name to handle messages linked to a "session"

Reply:
would you mind strong typing it?

Reply:
I can

Reply:
Done. Actually, it comes from gatling and it's supposed to be a Long.